src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -92,11 +92,21 @@ set_target_properties(System.Globalization.Native-Static PROPERTIES OUTPUT_NAME
 install (TARGETS System.Globalization.Native-Static DESTINATION .)
 
 if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
-    add_custom_command(TARGET System.Globalization.Native POST_BUILD
-        COMMENT "Verifying System.Globalization.Native.so dependencies"
-        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/../verify-so.sh
-            $<TARGET_FILE:System.Globalization.Native>
-            "Verification failed. System.Globalization.Native.so has undefined dependencies. These are likely ICU APIs that need to be added to icushim.h."
-        VERBATIM
-    )
+    if (BUILD_GLOBALIZATION_SHARED_LIBRARY)
```

Comment:
Other existing shim CMakefiles use `GEN_SHARED_LIB` for this purpose. Do the same for consistency?

https://github.com/dotnet/runtime/search?utf8=%E2%9C%93&q=GEN_SHARED_LIB&type=